lib/node/Creator.js

```diff
@@ -4,45 +4,64 @@
   information on nodes in mathJS
 */
 
-const math = require('mathjs');
 const NodeType = require('./Type');
 
+const opFns = {
+  '+': 'add',
+  '-': 'subtract',
+  '*': 'multiply',
+  '/': 'divide',
+  '^': 'pow',
+};
+
+const operatorNode = (op, fn, args) => {
```

Comment:
Eventually I'd want to move node creation functions into a separate package for math-ast tools.  The benefit of having tools such as node creation, traversal, transform, etc. methods separate from the parser is that people may want to build different parsers but using the same AST and related tools.

Before that though, I'd like to transition mathsteps to using the math-ast nodes instead of converting those nodes to mathjs-like nodes as we're doing now.